src/pages/Components/Notification/code.md

```diff
@@ -7,26 +7,36 @@ order: 2
 
 ```css
 @import 'settings-tools/_all-settings';
-@import '...';
+@import 'components/_c.notification';
 ```
 
 ## Basic usage
 
-Apply the classes...
+Wrap a h3, p and a a or button tags in a div and apply the `mc-notification` class:
 
 ```html
-<div class="class">Label</div>
+  <div class="mc-notification">
+    <h3>Notification title</h3>
+    <p>
+      Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa.
+    </p>
+    <a href="#">Link</a>
+  </div>
 ```
 
-<preview path="src/pages/Components/Notification/Notification/previews/Notification"></preview>
+<preview path="src/pages/Components/Notification/previews/notification-danger"></preview>
 
-## Variations
+## States
 
-use the same variations in the same order as the index page
+You have 4 states of notification available (self-explanatory): *information*(default), *success*, *warning* and *danger*. Apply them using the corresponding class: `mc-notification-[type]`.
 
-## Behaviors
+<preview path="src/pages/Components/Notification/previews/notification-states"></preview>
 
-use the same Behaviors in the same order as the index page
+## Variations
+
+Using `mc-notification-closable` class you have a closable notification:
+
+<preview path="src/pages/Components/Notification/previews/notification-variations"></preview>
 
 ## Responsive behaviors
```

Comment:
missing content here ?

Reply:
We gonna tackle this issue tomorrow in the polling.